api/src/main/java/org/openmrs/module/kenyaemr/advice/SyncHFEAppointmentsWithBahmniModule.java

```diff
@@ -552,4 +566,13 @@ private void processMCHEncounter(Encounter enc) throws Throwable {
 
         }
     }
+    
+    private String getAppointmentServiceUuidFromConcept( int conceptId) {
```

Comment:
how about drug refill?

Reply:
Drug refill we have explicitly handle its creation and editing separately within the code